packages/utils/tool-utils/src/snapshotNormalizer.ts

```diff
@@ -92,30 +92,55 @@ function getSortedBlobContent(content: string): string {
  * Helper function that normalizes the given snapshot tree. It sorts objects and arrays in the snapshot. It also
  * normalizes certain blob contents for which the order of content does not matter. For example, garbage collection
  * blobs contains objects / arrays whose element order do not matter.
+ * We also promote any .channels subtrees, and exclude the root .metadata blob for backwards compatibility.
```

Comment:
I am not sure if this is needed. The snapshot tests should work without this. All snapshots written by the tests should be in the new format so they should be comparable.
The "writes snapshot in correct format" tests should fail and that means you need to update the latest snapshot to be of this new format.

Reply:
Removed it.